edn/build.rs

```diff
@@ -0,0 +1,6 @@
+
+extern crate peg;
+
+fn main() {
+    peg::cargo_build("src/edn.rustpeg");
```

Comment:
Could you add `.gitattributes` and a modeline to have the `.rustpeg` file marked as Rust?

Details:

https://github.com/github/linguist/blob/master/README.md#using-gitattributes